js/lib/Locale.js

```diff
@@ -578,6 +578,212 @@ Locale.a1toa3langmap = {
     "zu": "zul"
 };
 
```

Comment:
I guess it's from https://unicode.org/iso15924/iso15924-codes.html 
How about leaving a link to where this data came from? (In case of a list change)

Reply:
Yes, it came from there originally. I copied it from js/data/locale/scripts.json ... I wanted to make sure that the Locale code could run without loading any data, which would require async operation in some situations. Requiring async operation for `new Locale` is not something I wanted to happen.

Anyways, yes, I'll put that link in the comment.